### PR TITLE
[Merged by Bors] - chore: uniformise more copyright headers

### PR DESCRIPTION
**Pull request by grunweg**
The comment in `Data/Finset/Sym` is a porting artifact and can just be removed.

Pre-requisite for rewriting the copyright header linter in Lean in #16574.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Finset/Sym.lean [hunk 1]
```diff
@@ -2,9 +2,8 @@
 Copyright (c) 2021 Yaël Dillies. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Yaël Dillies
-
-[`data.finset.sym`@`98e83c3d541c77cdb7da20d79611a780ff8e7d90`..`02ba8949f486ebecf93fe7460f1ed0564b5e442c`](https://leanprover-community.github.io/mathlib-port-status/file/data/finset/sym?range=98e83c3d541c77cdb7da20d79611a780ff8e7d90..02ba8949f486ebecf93fe7460f1ed0564b5e442c)
 -/
+
```
Comment (YaelDillies) [inline Mathlib/Data/Finset/Sym.lean:6]: I think we tend to *not* have the line break

**Review by YaelDillies — COMMENTED** [review]
I have no idea what this header is. Looks like it is a porting artifact. #2168 is the porting PR.

**Comment by grunweg** [comment]
In other words, that header can be removed, right? Thanks, that what I wanted to know.

File: Mathlib/Data/Finset/Sym.lean (hunk 1 above)
Comment (grunweg) [reply Mathlib/Data/Finset/Sym.lean:6]: Right; let me fix that quickly.

Comment (grunweg) [reply Mathlib/Data/Finset/Sym.lean]: To be more precise: the style guidelines advice against it, and mathlib is sufficiently un-ambiguous that this PR is not the place to litigate the question.

**Review by YaelDillies — APPROVED** [review]
maintainer merge

**Comment by grunweg** [comment]
Thanks for the fast look!

**Comment by bryangingechen** [comment]
Thanks!
bors r+

**Pull request closed without merging**